docs/conditional-artifacts-parameters.md

```diff
@@ -51,8 +50,8 @@ under step/DAG level output parameter. Both use the
               expression: "steps['flip-coin'].outputs.result == 'heads' ? steps.heads.outputs.result : steps.tails.outputs.result"
 ```
 
-* [Steps parameter example](https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/conditional-parameters.yaml)
-* [DAG parameter example](https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/dag-conditional-parameters.yaml)
+* [Steps parameter example](https://github.com/argoproj/argo-workflows/blob/master/examples/conditional-parameters.yaml)
+* [DAG parameter example](https://github.com/argoproj/argo-workflows/blob/master/examples/dag-conditional-parameters.yaml)
```

Comment:
Any reason we are moving away from raw content?

Reply:
I felt like the code formatting looked better linking to GitHub UI. Happy to revert!